### PR TITLE
CPDRP-799 feature flags and emails for beta

### PR DESCRIPTION
**Pull request by tonyheadford**
### Context
[Jira](https://dfedigital.atlassian.net/browse/CPDRP-799)
Enable feature flags and email participants for private beta of participant validation

### Changes proposed in this pull request
Mailers for ECTs and Mentors (FIP)
Add method to `InviteSchools` to activate `:participant_validation` feature flag for each school in a list of URNs provided by the caller and send the appropriate email to the school's recipients.
Add UTM campaigns for `participant-validation-beta` and `participant-validation-research`

### Review Checks
- [ ] All pages have automated accessibility checks via cypress
- [ ] All pages have visual tests via cypress + percy
- [ ] All `School` queries are correctly scoped - `eligible` when they need to be

### CODE REVIEW

File: app/services/invite_schools.rb [hunk 1]
```diff
@@ -230,6 +230,34 @@ def send_year2020_invite_email
...
+            start_url: start_url,
+            user_research_url: user_research_url,
+          ).deliver_later
+        elsif profile.mentor?
+          SchoolMailer.participant_validation_fip_mentor_email(
+            recipient: profile.user.email,
+            school_name: school.name,
+            start_url: start_url,
+          ).deliver_later
+        end
```
Comment (kicferk1) [inline app/services/invite_schools.rb:254]: Should we check that the school is actually doing FIP before we send this emails? CIP schools have mentors too, couple hundreds of them.

Comment (kicferk1) [reply app/services/invite_schools.rb:254]: Or should we perhaps make the email work with both cip and fip mentors? Probably a product question which one we want (or maybe we want something else).

**Review by kicferk1 — COMMENTED** [review]
I'm slightly worry about naming - if the mentor emails are just for FIP mentors, then we should make sure we don't send them to CIP mentors (or resolve this in some other way).

**Comment by ethanmills** [comment]
Looks like the template isn't set up quite correctly, but the mentor email will also have a UR link

File: app/services/invite_schools.rb (hunk 1 above)
Comment (tonyheadford) [reply app/services/invite_schools.rb:254]: Not sure how much of an issue it is at this point as this is to email a pre-selected set of FIP schools for the beta, but generally we should do this.

**Comment by tonyheadford** [comment]
> Looks like the template isn't set up quite correctly, but the mentor email will also have a UR link. I've messaged @Lloyd-K

Thanks, yes I did wonder, I was going to ask her this morning

**Comment by ethanmills** [comment]
> > Looks like the template isn't set up quite correctly, but the mentor email will also have a UR link. I've messaged @Lloyd-K
> 
> Thanks, yes I did wonder, I was going to ask her this morning

@tonyheadford Looks like it's been updated

File: app/services/invite_schools.rb (hunk 1 above)
Comment (tonyheadford) [reply app/services/invite_schools.rb]: Added a check for this just to be on the safe side.

**Review by kicferk1 — APPROVED** [review]

**Pull request closed without merging**